### DISCUSSION TITLE
Disable launch url on double click?

### DISCUSSION THREAD

**Discussion by waynesbrain** [keepassxreboot/keepassxc#11325] [Q&A]
I don't like the feature of launching on double click of the URL field. Is there a way to turn it off?

The problem for me is that I want to see the URL in the listing, but then I have to be careful where I click when I want to edit an item otherwise it launches the URL.

I would prefer to NEVER automatically launch any URL when I double click and to ALWAYS just edit the entry.

**Comment by droidmonkey** [comment]
Created a request: #11327

**Reply by waynesbrain** [reply]
Thanks, hope it’s not a bother
